build/release/Makefile

```diff
@@ -100,6 +114,27 @@ endif
 	fi
 
 # ====================================================================================
+# docs
+
+build.docs:
+
+	# checkout a fresh copy of the rook.github.io directory
```

Comment:
nit: remove the comment or make sure the `#` is on column 0 or else it will be part of a receipe